internal/test/go.mod

```diff
@@ -2,13 +2,19 @@ module github.com/openshift/windows-machine-config-operator/internal/test
 
 go 1.12
 
+replace (
```

Comment:
If you ran a command to do this, please mention that in the commit.

Reply:
No, I just manually added them